tests/kafkatest/services/performance/end_to_end_latency.py

```diff
@@ -36,17 +38,23 @@ def __init__(self, context, num_nodes, kafka, topic, num_records, consumer_fetch
             'num_records': num_records,
             'consumer_fetch_max_wait': consumer_fetch_max_wait,
             'acks': acks,
-            'kafka_opts': self.security_config.kafka_opts
+            'kafka_opts': self.security_config.kafka_opts,
+            'message_bytes': EndToEndLatencyService.MESSAGE_BYTES
         }
 
-    def _worker(self, idx, node):
-        args = self.args.copy()
-        self.security_config.setup_node(node)
+        for node in self.nodes:
+            node.version = version
+
+    @property
+    def security_config_file(self):
         if self.security_config.security_protocol != SecurityConfig.PLAINTEXT:
             security_config_file = SecurityConfig.CONFIG_DIR + "/security.properties"
-            node.account.create_file(security_config_file, str(self.security_config))
         else:
             security_config_file = ""
+        return security_config_file
+
+    def start_cmd(self, node):
+        args = self.args.copy()
         args.update({
             'zk_connect': self.kafka.zk.connect_setting(),
             'bootstrap_servers': self.kafka.bootstrap_servers(self.security_config.security_protocol),
```

Comment:
Line 61:
I think we want `'security_config_file': self.security_config_file`? `security_config_file` looks like it's out of scope now